spec.bs

```diff
@@ -767,6 +773,12 @@ The user agent MAY utilize the information in
 into a language and using locale-based formatting consistent with that of the
 website.
 
+If {{SecurePaymentConfirmationRequest/showOptOut}} is `true`, the user agent
+MUST give the user the opportunity to indicate that they want to opt out of the
+process. If the user indicates that they wish to opt-out, then the user agent
+must reject the {{PaymentRequest/show|show()}} promise with an
+"{{OptOutError}}" {{DOMException}}. See [[#sctn-user-opt-out]].
```

Comment:
Nit; let's be a little more specific about what they opt out from:

'indicate that they want to opt out of the process for the given relying party'.

(Maybe linking given relying party to `rpId`?)

Reply:
Done